ui/lib/apps/KeyViz/components/KeyViz.less

```diff
@@ -141,8 +141,7 @@
 
   .heatmap {
     flex-grow: 1;
-    margin: @padding-page;
-    margin-top: -32px;
+    margin: -32px @padding-page @padding-page;
```

Comment:
In my personal opinion, I prefer the original expression, 3 values expression makes me confused.

Reply:
I don't think so, but it's OK for me. I will revert it.